src/Root.cc

```diff
@@ -217,9 +234,8 @@ Errors Root::LoadSdfString(const std::string &_sdf)
 Errors Root::LoadSdfString(const std::string &_sdf, const ParserConfig &_config)
 {
   Errors errors;
-  SDFPtr sdfParsed(new SDF());
-  init(sdfParsed);
-
+  // Make a copy of the cached structure
+  SDFPtr sdfParsed = std::make_shared<SDF>(*cache.Access().sdfParsed);
```

Comment:
Also from the style guide
> Dynamic initialization of static local variables is allowed (and common)


```suggestion
  static gz::utils::NeverDestroyed<SdfCache> cache;
  SDFPtr sdfParsed = std::make_shared<SDF>(*cache.Access().sdfParsed);
```